src/content/blog/legacy-projects-with-unsupported-php-and-mysql-using-ddev.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Using DDEV to spin up a legacy PHP application"
+pubDate: 2025-05-27
+modifiedDate: 2025-05-27
+summary: How to use DDEV with outdated PHP and MySQL versions
+author: Garvin Hicking
+featureImage:
+  src: /img/blog/2025/05/museums-victoria-Di7WfLcrJ_I-unsplash.jpg
+  alt: Legacy computer museum
+  credit: "Photo by [Museums Victoria](https://unsplash.com/@museumsvictoria?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash) on [Unsplash](https://unsplash.com/photos/gray-mechanical-machine-lot-beside-wall-Di7WfLcrJ_I?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash)"
```

Comment:
I think stuff like `?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash` is not needed.